sdk/src/types/api/core/response.rs

```diff
@@ -117,20 +118,62 @@ pub struct SubmitBlockResponse {
     pub block_id: BlockId,
 }
 
-/// Describes the ledger inclusion state of a transaction.
+/// Describes the state of a block.
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 #[cfg_attr(
     feature = "serde",
     derive(serde::Serialize, serde::Deserialize),
     serde(rename_all = "camelCase")
 )]
-pub enum LedgerInclusionState {
-    Conflicting,
-    Included,
-    NoTransaction,
+pub enum BlockState {
+    // Stored but not confirmed or contains not yet included transaction.
```

Comment:
Isn't `or contains not yet included transaction` kinda obvious? It's not in the TIP desription

Reply:
The TIP has `If 'pending', the block contains not yet included transaction.`

Reply:
For TxState yeah, not for BlockState

Reply:
No, the other way around